_docs/11-training.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: splash
+title: "Training"
+permalink: /docs/training/
+hidden: true
+header:
+  overlay_color: "#5e616c"
+  overlay_image: /assets/images/201106-CORP-BNR-Design-Centers-FPGAs-and-plds-Banner-2880x280.jpg
+excerpt: "Welcome to the Training page."
+last_modified_at: 2018-01-10T11:22:01-05:00
+---
+
+# Training Material
+## Below is a list of training available on Youtube.
```

Comment:
"Below is a list of training available on Youtube." -> Training videos on YouTube

Reply:
Updated in latest commit.